### PR TITLE
Feat/cleanup readme

### PR DESCRIPTION
**Pull request by JoeyStewart**
What I did:

ReadMe and some conditionals for rendering in sort.

Did this break anything?
- [x] No
- [ ]  Yes
Type of change
- [x] New feature (non-breaking change which adds functionality)
- [x]  Styling
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.
Checklist:
- [ ]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [x]  There are comments where clarification/ organization is needed
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing
What's next?

### CODE REVIEW

File: src/Modal/DesignationSort.js [hunk 1]
```diff
@@ -14,13 +14,13 @@ const DesignationSort = ({ getUniqueDesignations, selectedDesignations, handleDe
           <div className="dropdown-toggle-des">Sort by Designation:</div>
           <div className="dropdown-menu-des">
             {getUniqueDesignations().map((designation) => (
-              <label key={designation} className="dropdown-item">
+              <label key={designation || "no-designation"} className="dropdown-item">
```
Comment (Jesuitman) [inline src/Modal/DesignationSort.js:17]: Excellently done

File: src/Modal/DesignationSort.js [hunk 2]
```diff
@@ -14,13 +14,13 @@ const DesignationSort = ({ getUniqueDesignations, selectedDesignations, handleDe
...
                 <input
                   type="checkbox"
                   defaultChecked={selectedDesignations.includes(designation)}
                   onChange={() => handleDesignationCheckboxChange(designation)}
                 />
-                {designation}
+                {designation || "No Designation"}
```
Comment (Jesuitman) [inline src/Modal/DesignationSort.js:23]: im really glad thi was such an easy fix

**Review by Jesuitman — APPROVED** [review]
good work on this! im really glad it was so fast

File: README.md [hunk 3]
```diff
@@ -90,6 +91,41 @@ Joey -  It was our first time working with a backend team, we needed to discuss
...
+### Backend Integration: 
+
+### Sprint Based development:
+
+### New Team Dyanmics with Backend Team:
+
+### Error Handling: 
+
+### Tailwind: 
+
```
Comment (Jesuitman) [inline README.md:106]: YEAAAAAAAHHHHHHH BOIIIIIIIIIIIII

**Review by Jesuitman — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -90,6 +91,41 @@ Joey -  It was our first time working with a backend team, we needed to discuss
...
+### Wins:
+Lex - This is a celebration of everything I have learned so far! I got to work with a backend team and learn more about what it takes to be a full stack dev.
+
+Joey - Getting the sort function to work was a massive win. I wanted it to be dynamic enough so that you could search for any park with any of the filter conditions provided. It turned out to be tougher of a task than I thought but the functionality of it turned out really nice.
+
+Arden - Wins would be finding useful libraries and documentation resources for our stretch, having strong collaboration within the frontend group, and integrating Tailwind quickly
+
+### Challenges: 
+Lex - pivoting from flutter and tailwind to just tailwind was disappointing but a needed step. I also felt greatly challenged by the weather modal because it had different rule than I had used before.
+
```
Comment (tenthwalker) [inline README.md:117]: I'll add my challenges in my branch since I'm still figuring out what to say for those.

File: src/Modal/StateSort.js [hunk 5]
```diff
@@ -2,15 +2,12 @@ import React, {useState} from "react";
 
 const StateSort = ({ states, selectedStateParks, handleStateCheckboxChange }) => {
   const [dropdownStateOpen, setDropdownStateOpen] = useState(false);
-  const [dropdownDesignationOpen, setDropdownDesignationOpen] = useState(false);
```
Comment (tenthwalker) [inline src/Modal/StateSort.js:5]: Thanks for cleaning this up!

**Review by tenthwalker — APPROVED** [review]
Thanks for getting those conditionals in!

**Pull request merged**